kopf/reactor/peering.py

```diff
@@ -227,3 +228,43 @@ def detect_own_id() -> str:
     now = datetime.datetime.utcnow().isoformat()
     rnd = ''.join(random.choices('abcdefhijklmnopqrstuvwxyz0123456789', k=6))
     return f'{user}@{host}/{now}/{rnd}'
+
+
+class PeerFactory:
```

Comment:
What I would suggest, is to consider using the `@classmethod` in `Peer` instead of creating the `PeerFactory` entity with `@staticmethods` (since any class is already a factory): e.g. `ourselves = Peer.detect(...)` — which can also return `None`. 

But since there is no hierarchy of peering classes, this is not critical.

Reply:
Good point, done.